lib/method_cache/proxy.rb

```
@@ -190,7 +190,7 @@ def write_to_cache(key, value)
         else
           cache.set(key, value, expiry(value))
         end
-      rescue StandardError, Dalli::DalliError => e
+      rescue StandardError, Dalli::ValueOverMaxSize => e
         if e.message =~ /Value too large/
```

Should this not be two separate rescues?  There's no need to check the e.message if it's split out?  Also, protects us from message changes by the gem.

Out of curiosity why would we want to swallow any errors here?

If all we were doing was swallowing the ValueOverMaxSize errors and now we want to know about them so we can control logic flow.. seems like we just remove the rescue entirely?

If we are trying to get it to swallow ValueOverMaxSize errors again just to get us working.. I think just changing it to ...
```
rescue Dalli::ValueOverMaxSize => e
  nil
end
```
would do it no?

---

I changed that after pushing for the review. I was lazy then my brain caught up to thinking. :)